src/box/vy_lsm.h

```diff
@@ -176,7 +176,7 @@ vy_lsm_env_destroy(struct vy_lsm_env *env);
  *   key, *augmented* with the parts of the primary key from the
```

Comment:
These are actually two distinct changes: you either have to split this into two commits for `box` and `salad` components, or just drop these changes.

Reply:
Dropped the commit.